tests/Sniffs/Exceptions/ExceptionNameEndsWithExceptionSniffTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Consistence\Sniffs\Exceptions;
+
+class ExceptionNameEndsWithExceptionSniffTest extends \Consistence\Sniffs\TestCase
+{
+
+	public function testInvalidClassName()
```

Comment:
`InvalidExceptionName`? + datafile rename (to be more specific and avoid clashing the data files)